jni/events/baseaudioevent.cpp

```diff
@@ -309,7 +309,7 @@ void BaseAudioEvent::mixBuffer( AudioBuffer* outputBuffer, int bufferPosition,
                 {
                     readPointer -= _sampleStart;
 
-                    for ( c = 0, ca = _buffer->amountOfChannels; c < ca; ++c )
+                    for ( c = 0; c < outputChannels; ++c )
                     {
```

Comment:
Caught bug that would cause a violent crash when mixing a buffer containing more channels than the target buffer (this would occur if the samples would be mixed when a loop starts)